src/_labs/TourPoint/TourPoint.story.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { forwardRef } from 'react';
+
+import { Story } from '@storybook/react';
+
+import { TourPoint } from './TourPoint';
+
+import { Header } from '../../typography';
+
+export default {
+  title: 'labs/TourPoint',
+  component: TourPoint,
+  argTypes: {
+    icon: { control: { disable: true } },
+    onClose: { control: { disable: true } },
+    target: { control: { disable: true } },
+  },
+  excludeStories: ['Card'],
+};
+
+type Props = any;
+
+new Image().src = 'http://placekitten.com/320/213';
```

Comment:
🔥 preloading the image into the browser cache 🔥 I like thattttttt 😮‍💨

Reply:
Copied it from the story Sean wrote!